### DISCUSSION TITLE
How to add a custom image block to BlockSuite editor in angular and add it to slash menu?

### DISCUSSION THREAD

**Discussion by bagabba** [toeverything/blocksuite#7768] [Q&A]
I have BlockSuite integrated to my angular application. In my usecase, I need provision to add an image/video to the editor via upload or URL from the slash menu. Similarly, the image/video has to be displayed in the block from the URL without an xhr call being fired (image using img src). Any sample code or guidance is appreciated. Thanks in advance!

**Comment by raintoway** [comment]
i suggestion u custom image block by lit instead of angular

**Reply by bagabba** [reply]
Any reference or documentation for the same?
